charts/isdargo/templates/configmaps/isd-feature-flag-config.yml

```diff
@@ -1,4 +1,5 @@
 {{- if (eq .Values.installationMode "OEA-AP") }}
+{{- if .Values.installkeycloak }}
```

Comment:
The if else condition should be for only or below flag i.e flag: true or not .. Please update this accordingly